sdcm/k8s_configs/eks/scylla-node-prepare.yaml

```diff
@@ -186,6 +186,14 @@ spec:
             value: /mnt/hostfs/mnt/raid-disks/disk0
           - name: SCYLLAD_CONF_MOUNT
             value: /mnt/scylla.d/
+        lifecycle:
+          postStart:
+            exec:
+              command:
+                - "/bin/bash"
+                - "-c"
+                - "--"
+                - "yum-config-manager --disable scylla --disable scylla-generic --disable scylladb-scylla-3rdparty"
```

Comment:
close to may suggestion building a new image that only removed the repo files.

but this one is a nice trick (should write it down in the back of my mind :))